net/miniupnpc/Portfile

```diff
@@ -14,11 +14,11 @@ description         Lightweight client for UPnP protocol
 long_description    ${description}
 
 homepage            http://miniupnp.free.fr/
-master_sites        http://miniupnp.free.fr/files/download.php?file=${distname}${extract.suffix}&dummy=
-
-checksums           rmd160  9cac1e3f7ed8ff60c1ed57644224eba83adbab81 \
-                    sha256  481a5e4aede64e9ef29895b218836c3608d973e77a35b4f228ab1f3629412c4b \
-                    size    102932
+master_sites        http://miniupnp.free.fr/files/
+#distname            ${name}-${version}
```

Comment:
please don't comment out no longer needed lines, just remove them.

Reply:
Sorry, this is just my mistake. Will fix it now.